src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -585,6 +585,7 @@ private void ValidateAndBuildHostJsonConfigurationIfFileExists(ScriptApplication
 
         private async Task PreRunConditions()
         {
+            CheckSettingFunctionWorkerRuntime();
```

Comment:
Why is this the first thing that needs to happen?

Reply:
@liliankasem when local.settings.json file is missing and if we start hosting the function app . we just get the error 
![image](https://github.com/user-attachments/assets/83edf3d7-8e45-4b3d-b84a-8b4e19739676)
To Avoid this  CheckSettingFunctionWorkerRuntime() needs to run first to add setting for Function_Worker_Runtime